### PR TITLE
docs(component-store): init, read, write sections

### PR DESCRIPTION
**Pull request by alex-okrushko**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

Sections for initializing, reading and updating ComponentStore.

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by brandonroberts** [comment]
@alex-okrushko are you planning to add content for the comparison and effects pages?

File: projects/ngrx.io/content/guide/component-store/install.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Installation
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/install.md:1]: I've not read the other documentation of this PR (for now).
I just wanted to mention that we also have `ng-add` support 🙂

**Comment by alex-okrushko** [comment]
> @alex-okrushko are you planning to add content for the comparison and effects pages?

@brandonroberts yes, I do. I thought it would be a bit too much for a single PR. Do you want me to add it here as a new commit or as a different follow-up PR?

**Comment by brandonroberts** [comment]
Follow-up is fine with me

File: projects/ngrx.io/content/guide/component-store/install.md (hunk 1 above)
Comment (alex-okrushko) [reply projects/ngrx.io/content/guide/component-store/install.md:1]: Nice.
Done.

**Pull request merged**